src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days > 6) {
+    return (days * 40) - 50;
```

Comment:
Good job! I have almost the same solution:)
Although you don't really need brackets, because multiplication will be performed before subtraction even without them.